woocommerce/woocommerce-gutenberg-products-block
# [Store Customization MVP] Verticals API client

## What
This PR adds a new `src/Verticals/Client.php` class (and tests) to connect to the Verticals API.
It allows has 2 functions:
- `get_verticals`: returns all the verticals with images.
- `get_vertical_images`: returns the images for a given vertical id.

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/10615

## Why
This is the second step required for the Store Customization MVP project, described on https://github.com/woocommerce/woocommerce-blocks/issues/10503

## Testing Instructions



Automated tests are in `tests/php/Verticals/ClientTest.php`.

1. Add this code somewhere in the plugin (for example on the `woocommerce-gutenberg-products-block.php`)
```
$verticals_api = new \Automattic\WooCommerce\Blocks\Verticals\Client();
$verticals     = $verticals_api->get_verticals();
var_dump( $verticals );
$images = $verticals_api->get_vertical_images( 3 );
var_dump( $images );
```
2. Go to the home page and check both requests are returning data.


## Screenshots or screencast

-

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:
